message/issue/experimental.md

```diff
@@ -0,0 +1,5 @@
+Hey @${author}! You seem to be running an experimental version (${version}) of ${repository_name} that should not be run on production systems.
+
+Plugin compatibility **will** break often on experimental systems, so unless you are a plugin developer assisting the next version of the API, please reproduce on the reccomended build or latest build of Sponge (${stable}).
+
+This was an automated post, if you strongly disagree with this bot's response please reply stating why.
```

Comment:
Your tone should be slightly weaker, e.g. `This was an automated post. If you have a reason to run on ${version}, please clarify.`

Reply:
tbh I was thinking of removing this part for consistency. After all, none of the bot messages has the "this is an automated post.." disclaimer.